recipes/wt/all/conanfile.py

```diff
@@ -90,106 +125,172 @@ def _configure_cmake(self):
+            self.cpp_info.components["wttest"].libs = [self._lib_name("wttest")]
+            self.cpp_info.components["wttest"].requires = ["wtmain"]
+        # wthttp
         if self.options.connector_http:
-            self.cpp_info.libs.append('wthttp')
-        if self.settings.os == 'Windows':
-            if self.options.connector_isapi:
-                self.cpp_info.libs.append('wtisapi')
-        else:
-            if self.options.connector_fcgi:
-                self.cpp_info.libs.append('wtfcgi')
-        self.cpp_info.libs.append('wt')
-        if self.settings.build_type == 'Debug':
-            self.cpp_info.libs = ['%sd' % lib for lib in self.cpp_info.libs]
-        if self.settings.os == 'Linux':
-            self.cpp_info.system_libs.append('dl')
-        elif self.settings.os == 'Windows':
-            self.cpp_info.system_libs.extend(['ws2_32', 'mswsock', 'wsock32'])
+            self.cpp_info.components["wthttp"].names["cmake_find_package"] = "HTTP"
+            self.cpp_info.components["wthttp"].names["cmake_find_package_multi"] = "HTTP"
+            self.cpp_info.components["wthttp"].libs = [self._lib_name("wthttp")]
+            self.cpp_info.components["wthttp"].requires = ["wtmain", "boost::boost", "zlib::zlib"]
+            if self.options.with_ssl:
+                self.cpp_info.components["wthttp"].requires.append("openssl::openssl")
+        # wtisapi
+        if self.options.get_safe("connector_isapi"):
+            self.cpp_info.components["wtisapi"].names["cmake_find_package"] = "Isapi"
+            self.cpp_info.components["wtisapi"].names["cmake_find_package_multi"] = "Isapi"
+            self.cpp_info.components["wtisapi"].libs = [self._lib_name("wtisapi")]
+            self.cpp_info.components["wtisapi"].requires = ["wtmain"]
+        # wtfcgi
+        if self.options.get_safe("connector_fcgi"):
+            self.cpp_info.components["wtfcgi"].names["cmake_find_package"] = "FCGI"
+            self.cpp_info.components["wtfcgi"].names["cmake_find_package_multi"] = "FCGI"
+            self.cpp_info.components["wtfcgi"].libs = [self._lib_name("wtfcgi")]
+            self.cpp_info.components["wtfcgi"].requires = ["wtmain"]
+            if self.options.with_ssl:
+                self.cpp_info.components["wtfcgi"].requires.append("openssl::openssl")
+        # wtdbo
+        if self.options.with_dbo:
+            self.cpp_info.components["wtdbo"].names["cmake_find_package"] = "Dbo"
+            self.cpp_info.components["wtdbo"].names["cmake_find_package_multi"] = "Dbo"
+            self.cpp_info.components["wtdbo"].libs = [self._lib_name("wtdbo")]
+            if self.options.get_safe("with_unwind"):
+                self.cpp_info.components["wtdbo"].requires.append("libunwind::libunwind")
+        # wtdbosqlite3
+        if self.options.get_safe("with_sqlite"):
+            self.cpp_info.components["wtdbosqlite3"].names["cmake_find_package"] = "DboSqlite3"
```

Comment:
Is lower case components a rules? We (since space has done a lot of the heavy light 🥇 ) have been very consistent on this... should it be a hook?

Reply:
Yes maybe. I think also that these names should be qualified (because of https://github.com/conan-io/conan/issues/7560).